dynamic-layers/virtualization-layer/recipes-extended/xen/xen-tb.inc

```diff
@@ -4,9 +4,9 @@ PV = "${XEN_REL}+tb"
 
 SRC_URI = "git://github.com/TrenchBoot/xen.git;branch=${XEN_BRANCH};protocol=https"
 
-SRCREV = "0b53c6a5b796f3b1a146f612fd06fa37d1bf7714"
+SRCREV = "61d5a16536ffc5a208879c23cd0315bb130f57d1"
 
 S = "${WORKDIR}/git"
 
 XEN_REL = "4.17"
-XEN_BRANCH = "aem_intel_fixes_v0.4.1"
+XEN_BRANCH = "aem-staging-14-10-24-rebase"
```

Comment:
Please don't use this branch, it isn't stable and will be overwritten. Since this PR is for Linux only, I think it would be better to keep the old revision, even though it won't boot. It is better to have just Linux instead of having both that will break as soon as I start next iteration of rebase.

Reply:
So let's just focus on testing Linux in this MR.

Reply:
Ok. I thought I tested it, but apparently not enough. I can help fixing xen in a followup, since we need Linux here.